python-first/get_first.py

```diff
@@ -4,20 +4,22 @@
 def get_first(iterable, value=None, key=None, default=None):
     match value is None, callable(key):
         case (True, True):
-            gen = (val for val in iterable if key(val))
+            gen = (elem for elem in iterable if key(elem))
         case (False, True):
-            gen = (val for val in iterable if key(val) == value)
+            gen = (elem for elem in iterable if key(elem) == value)
         case (True, False):
-            gen = (val for val in iterable if val)
+            gen = (elem for elem in iterable if elem)
         case (False, False):
-            gen = (val for val in iterable if val == value)
+            gen = (elem for elem in iterable if elem == value)
```

Comment:
I think I find `elem` or even `element` a better name for each element in iterable than juggling `val` and `value`. What do you think?